apps/tinyVario/app.js

```diff
@@ -266,6 +266,9 @@ function initAltMenu() {
         }},
         {type:"btn", font:"16%", pad:1, fillx:1, label:"SAVE", cb: ()=>{
           require('Storage').writeJSON("tinyVario.json", settings);
+          var o=Bangle.getOptions();
```

Comment:
You don't actually need this (and it's probably better not to use it) - just do `Bangle.setOptions({seaLevelPressure:qnh});`

Reply:
Thanks!